osu.Game/Localisation/CommonStrings.cs

```diff
@@ -99,6 +99,31 @@ public static class CommonStrings
         /// </summary>
         public static LocalisableString Description => new TranslatableString(getKey(@"description"), @"Description");
 
+        /// <summary>
+        /// "Always"
+        /// </summary>
+        public static LocalisableString Always => new TranslatableString(getKey(@"always"), @"Always");
+
+        /// <summary>
+        /// "Never"
+        /// </summary>
+        public static LocalisableString Never => new TranslatableString(getKey(@"never"), @"Never");
+
+        /// <summary>
+        /// "None"
+        /// </summary>
+        public static LocalisableString None => new TranslatableString(getKey(@"none"), @"None");
+
+        /// <summary>
+        /// "Full"
+        /// </summary>
+        public static LocalisableString Full => new TranslatableString(getKey(@"full"), @"Full");
+
+        /// <summary>
+        /// "Off"
+        /// </summary>
+        public static LocalisableString Off => new TranslatableString(getKey(@"off"), @"Off");
```

Comment:
Unconvinced that strings like this are going to commonise very well. Some languages have a notion of gendered nouns, which also sometimes causes adjectives to have gendered forms, too, and so "off" becomes context-sensitive. French and Polish do this.

Reply:
I see, yeah, you are right, I will move it to separate localisation classes.